core/src/services/gdrive/core.rs

```diff
@@ -399,15 +411,9 @@ impl GdriveCore {
 
         let url = format!("https://www.googleapis.com/drive/v3/files/{}", file_id);
 
-        let source_abs_path = build_abs_path(&self.root, source);
-        let mut source_parent: Vec<&str> = source_abs_path
-            .split('/')
-            .filter(|&x| !x.is_empty())
-            .collect();
+        let mut source_parent: Vec<&str> = source.split('/').filter(|&x| !x.is_empty()).collect();
```

Comment:
maybe the missing `build_abs_path(&self.root, source)` cause the `unwrap()` on a `None` value in line 426